docs/source/user_guide.md

```diff
@@ -2,64 +2,70 @@
-
-The second is to create custom manifest files in order to satisfy security organizations' requirements. These files contain the package name, download URL, and sha256 hash of all packages and dependencies from the input conda environment file. Currently Iron Bank is the only format supported, but others can be easily added by modifying [this file](https://github.com/MetroStar/conda-vendor/blob/main/conda_vendor/custom_manifest.py).
-
+The tool works by intaking a [conda environment yaml file](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#create-env-file-manually) and performing a conda solve to determine what packages are needed.  Conda vendor will download those packages and provide `repodata.json`
+files for the relevant directories, so that the downloaded files are a
+valid conda channel.
 
 ## Usage
 
-Conda-vendor has two main steps to create a local channel. First, a meta-manifest file is created as an intermediate artifact. With an existing meta-manifest file, a local conda channel can then be created.
-
-The intermediate meta-manifest is generated to allow for the creation of custom software manifests. These manifests can then be used obtain package approval from an organization's cybersecurity team.
-
-### Creating a Meta-manifest
-
-Conda-vendor solves an environment with conda from an `environment.yaml` and determines all the packages that are required. The metadata for these required packages is stored in a file called `meta_manifest.yaml`. To create this file, run:
-
-	conda vendor meta-manifest --environment-yaml environment.yaml
-		
-The above command will output a `meta_manifest.yaml` file in the current directory. 
-
-### Creating a Local Channel
-
-With a meta-manifest file created, conda-vendor can then create local channels. 
-	
-	conda vendor channels --meta-manifest-path ./meta_manifest.yaml
-
-This will create a directory called `local_channel` that will contain the same number of channels as were listed in the original `environment.yaml` file. These local channels will only contain the packages that are needed to satisfy the solved environment from the `meta-manifest` step.
-
-### Using the Local channel
+#### Creating a Local Channel
+The most basic way to use `conda-vendor` is to just pass an `environment.yaml`
+file to the `vendor` subcommand
+```bash
+$ conda-vendor vendor --file my_environment.yaml
+```
+
+This will solve the environment specified in `my_environment.yaml` using the
+host platform.  Sometimes it is necessary or desirable to solve for another
+platform.  The `vendor` command allows this with the `--platform` flag.  The usage looks like
+```bash
+$ conda-vendor vendor --file my_environment.yaml  --platform linux-64
+```
+
+By default `conda-vendor` will use a set of *virtual packages* for the specified
```

Comment:
Are "virtual packages" well understood by the conda community?  It is unclear to me what is going on here.

Reply:
fair question.  my guess is, no, they aren't.  I don't really think conda vendor is the place to teach everyone, though.  maybe a link.  since virtual packages are sort of the key to the whole thing.

for your info:
https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-virtual.html

Reply:
yeah, that link is perfect

Reply:
added more verbiage around virtual packages.